mutagen/oggvorbis.py

```diff
@@ -68,6 +68,8 @@ def __init__(self, fileobj):
                 "page contains a packet too short to be valid")
         (self.channels, self.sample_rate, max_bitrate, nominal_bitrate,
          min_bitrate) = struct.unpack("<B4i", page.packets[0][11:28])
+        if self.sample_rate < 1:
+            raise OggVorbisHeaderError("sample rate can't be negative")
```

Comment:
The error message doesn't match the check. Assuming the unsigned thing is fixed I'd say "can't be zero".